stage0/supply-chain/imports.lock

```diff
@@ -0,0 +1,72 @@
+
+# cargo-vet imports lock
```

Comment:
Was this manually created? If no, perhaps add some instructions for updating it, for future reference.

Reply:
No, `cargo vet` maintains it itself.